Android.bp

```diff
@@ -0,0 +1,170 @@
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package {
+    default_applicable_licenses: ["external_intel_libva_license"],
+}
+
+license {
+    name: "external_intel_libva_license",
+    visibility: [":__subpackages__"],
+    license_text: [
+        "COPYING",
+    ],
+}
+
+sh_binary_host {
+    name: "libva_gen_version_script",
+    src: "build/gen_version.sh",
+}
+
+genrule {
+    name: "libva_gen_version",
+    srcs: [
+        "configure.ac",
+        "va/va_version.h.in",
+    ],
+    tools: [
+        "libva_gen_version_script",
+    ],
+    out: ["va/va_version.h"],
+    cmd: "$(location libva_gen_version_script) --configure_ac $(in) > $(out)",
+}
+
+cc_library_headers {
+    name: "libva_headers",
+
+    export_include_dirs: [
+        ".",
+        "va",
+        "va/drm"
+    ],
+
+    generated_headers: [
+        "libva_gen_version",
+    ],
+    export_generated_headers: [
+        "libva_gen_version",
+    ],
+
+    vendor: true,
+    enabled: false,
+    arch: {
+        x86_64: {
+            enabled: true,
+        },
+    },
+}
+
+cc_library_shared {
+    name: "libva",
+
+    shared_libs: [
+        "libdl",
+        "libdrm",
+        "libcutils",
+        "liblog",
+    ],
+
+    local_include_dirs: [
+        ".",
+        "va"
+    ],
+
+    generated_headers: [
+        "libva_gen_version",
+    ],
+    export_generated_headers: [
+        "libva_gen_version",
+    ],
+
+    header_libs: [
+        "libutils_headers",
+    ],
+
+    srcs: [
+        "va/va.c",
+        "va/va_trace.c",
+        "va/va_str.c",
+        "va/drm/va_drm.c",
+        "va/drm/va_drm_auth.c",
+        "va/drm/va_drm_utils.c",
+    ],
+
+    cflags: [
+        "-Werror",
+        "-Wno-error",
+        "-DSYSCONFDIR=\"/vendor/etc\"",
+        "-DLOG_TAG=\"libva\"",
+    ],
+
+    arch: {
+        x86_64: {
+            cflags: ["-DVA_DRIVERS_PATH=\"/vendor/lib64\""],
+            enabled: true,
+        },
+    },
+
+    vendor: true,
+    enabled: false,
+}
+
+cc_library_shared {
+    name: "libva-android",
+
+    static_libs: [
+        "libarect",
+    ],
+
+    shared_libs: [
+        "libva",
+        "libdrm",
+        "liblog",
+    ],
+
+    local_include_dirs: [
+        "va",
+        "va/drm",
+    ],
+
+    header_libs: [
+        "libnativebase_headers",
+        "libutils_headers",
+    ],
+
+    srcs: [
+        "va/android/va_android.cpp",
+        "va/drm/va_drm_utils.c",
+    ],
+
+    cflags: [
+        "-Wall",
+        "-Werror",
+        "-Wno-error",
+    ],
```

Comment:
You may also want to add `"-DLOG_TAG=\"libva-android\""` to match the [Android.mk](https://github.com/intel/libva/blob/01153523f25ac703e7ce2afc0023788a90f6c1bb/va/Android.mk#L90).